### ISSUE TITLE
Focus topic input and open typeahead for new topics

### ISSUE THREAD

**Issue by alya** [zulip/zulip-mobile#4875]
As a follow up to #4773 / #4764, we should do the following when starting a new topic:

1. Put the cursor in the topic field.
2. Automatically pop up the typeahead. This may come for free with (1).

**Comment by gnprice** [comment]
This is for the experience when you're composing a new message. When implementing this, we should check that when you're *editing* a message instead, the focus still goes first to the message-content input.

**Comment by alya** [comment]
It looks like we have this already? Should this issue be closed?

**Comment by chrisbobbe** [comment]
> It looks like we have this already? Should this issue be closed?

IIUC, we got the compose-new-message part recently, in e21e7dd94 (released in v27.170). I'm not sure about the edit-message behavior that Greg says we should check. From briefly reading the code, I suspect that's still working as intended, but it'd be good to check. Would you mind checking, Alya?

**Comment by alya** [comment]
Yep, the focus for editing is still on the message, at least on Android.
